src/selectors/account$.js

```diff
@@ -21,27 +57,30 @@ export const activeAccounts$ = createSelector(
 
 export const accountsArray$ = createSelector(
     accounts$,
-    hasContent$,
+    (accounts) => [...accounts.values()]
+);
+
+export const legacyAccounts$ = createSelector(
+    accountsArray$,
+    (accounts) => accounts.filter(account => account.isLegacy)
+);
+
+export const activeAccountsArray$ = createSelector(
+    accountsArray$,
```

Comment:
The `activeAccountsArray$` could also just be a `[...activAccounts$.values()]`, just like `accountsArray$` is just an array of `accounts$.values()`.